tests/Suites/Feature/Schema/SchemaFactoryTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Illuminate\Database\Connectors\ConnectionFactory;
+use Illuminate\Database\Schema\MySqlBuilder;
+use Illuminate\Database\Schema\PostgresBuilder;
+use Illuminate\Database\Schema\SQLiteBuilder;
+use Illuminate\Database\Schema\SqlServerBuilder;
+use Illuminate\Support\Facades\DB;
+use Plank\Snapshots\Contracts\ManagesCreatedTables;
+use Plank\Snapshots\Contracts\ManagesVersions;
+use Plank\Snapshots\Factory\SchemaBuilderFactory;
+use Plank\Snapshots\Schema\MySqlSnapshotBuilder;
+use Plank\Snapshots\Schema\PostgresSnapshotBuilder;
+use Plank\Snapshots\Schema\SnapshotBuilder;
+use Plank\Snapshots\Schema\SQLiteSnapshotBuilder;
+use Plank\Snapshots\Schema\SqlServerSnapshotBuilder;
+
+describe('The Schema Builder is resolved in the container based on the db connection', function () {
```

Comment:
Not something we need to tackle in this PR: It might be interesting to eventually have the test suites actually run on each schema builder, and see if they behave as expected. 

This is a great first step towards that imho, just making sure the system can resolve correct builder is excellent.

Reply:
Would definitely be nice assurance.